### PR TITLE
check wallet key on endpoint

### PR DESCRIPTION
**Pull request by talvasconcelos**
Check for wallet key on API endpoint.

### CODE REVIEW

**Comment by talvasconcelos** [comment]
@prusnak I introduced your changes on this one!

File: views_api.py [hunk 1]
```diff
@@ -275,12 +276,18 @@ async def api_check_fiat_rate(currency):
 
 ## ITEMS
 @tpos_ext.put("/api/v1/tposs/{tpos_id}/items", status_code=HTTPStatus.CREATED)
-async def api_tpos_create_items(data: CreateUpdateItemData, tpos_id: str):
+async def api_tpos_create_items(
+    data: CreateUpdateItemData,
+    tpos_id: str,
+    wallet: WalletTypeInfo = Depends(require_invoice_key),
```
Comment (motorina0) [inline views_api.py:282]: shouldn't this be an `require_admin_key` because it is a write operation?

**Comment by prusnak** [comment]
> @prusnak I introduced your changes on this one!

I see, thanks. Will you also address this one? https://github.com/lnbits/tpos/pull/34

File: views_api.py (hunk 1 above)
Comment (talvasconcelos) [reply views_api.py:282]: ok

**Review by prusnak — APPROVED** [review]

**Review by motorina0 — APPROVED** [review]

**Review by prusnak — CHANGES_REQUESTED** [review]
After commit bd07dcbcac03b20bbc9d939e4be2d7e68d100ea5 I can no longer import JSON with items (401 unauthorized)

**Review by prusnak — APPROVED** [review]
I confirm that d7908f4 fixes the JSON import issue 🎉

**Comment by talvasconcelos** [comment]
can we merge it?

**Comment by prusnak** [comment]
> can we merge it?

Merge ACK. Please also do a release. We'll do a PR to lnbits-extensions repo afterwards.

**Pull request merged**